database/java/2017/15089.yaml

```diff
@@ -0,0 +1,41 @@
+---
+cve: 2017-15089
+title: CVE in org.infinispan:infinispan-client-hotrod
+description: >
+    It was found that the Hotrod client in Infinispan before 9.2.0.CR1 would unsafely read deserialized data on information from the cache. An authenticated attacker could inject a malicious object into the data cache and attain deserialization on the client, and possibly conduct further attacks.
+cvss_v2: 6.5
+references:
+    - http://www.securitytracker.com/id/1040360
+    - https://access.redhat.com/errata/RHSA-2018:0294
+    - https://github.com/infinispan/infinispan/pull/5639
+
+affected:
+    - groupId: org.infinispan
+      artifactId: infinispan-client-hotrod
+      version:
+        - "<=9.2.0.CR1"
```

Comment:
This is actually the first version which contains a fix for this vulnerability. Previous (affected) version is [`9.2.0.Beta2`](https://mvnrepository.com/artifact/org.infinispan/infinispan-client-hotrod) so this should be `"<=9.2.0.Beta2"`.